opentelemetry-sdk/src/opentelemetry/sdk/logs/__init__.py

```diff
@@ -111,6 +112,48 @@ def force_flush(self, timeout_millis: int = 30000):
         """
 
 
+class OTLPHandler(logging.Handler):
+    """A handler class which writes logging records, in OTLP format, to
+    a network destination or file.
+    """
+
+    def __init__(self, level=logging.NOTSET, log_emitter=None) -> None:
+        super().__init__(level=level)
+        self._log_emitter = log_emitter or get_log_emitter(__name__)
+
+    def _translate(self, record: logging.LogRecord) -> LogRecord:
+        timestamp = int(record.created * 1e9)
+        span_context = get_current_span().get_span_context()
+        # TODO: attributes (or resource attributes?) from record metadata
```

Comment:
FYI, I believe attributes can be populated from both custom and resources.

Reply:
So `logging.LogRecord` https://docs.python.org/3/library/logging.html#logrecord-attributes has a number of attributes like funcName, filename, and processName etc.. I was wondering if we should just make these as OTLP log record attributes https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md#field-attributes or log record resource attributes.

Reply:
For `logging.LogRecord` attributes, it makes more sense to be part of the OTLP log record attributes.

When I was saying "custom" attributes, I meant the ability for users to add custom attributes to the `LogRecord` themselves (it is a feature in OpenCensus as well), but may not be defined yet in the specs.

Not any concern for now, just something to be mindful of.

Reply:
`logging.LogRecord` has an (optional) `extra` attribute that holds arbitrary user-injected attributes.
This is not super well documented, but the docs can be found a couple paragraphs down from [here](https://docs.python.org/3/library/logging.html#logging.Logger.debug).

From a users perspective, it looks like this:

```python
logging.debug("msg", extra={...})
```

Would it be possible to map this to OTLP log record attributes? You could take the top-level attributes of `extra` (which would be my preference), or require them to be nested under a specific key (e.g. `logging.debug("msg", extra={"oltp-attributes": {...}})`).